### ISSUE TITLE
PFQueryTableViewController && Parse.enableLocalDatastore()

### ISSUE THREAD

**Issue by flashspys** [ParsePlatform/ParseUI-iOS#26]
I'm using `PFQueryTableViewController` in Swift and I have following problem:

When enabling localDatastore the  `objectsDidLoad` method will never be called. After disabling LocalDatastore everything works fine.
Does other ones have the same problem ?

**Comment by nlutsenko-fb** [comment]
Hey @flashspys,

I can't seem to reproduce the problem. Any chance you can provide a repro here?
(feel free to reopen if you still experience the problem)

**Comment by flashspys** [comment]
I've tried it and the error still occurs. Here is my Repo: https://github.com/flashspys/Cocoaheads-Swift You can uncomment in AppDelegate on line 23 the line `Parse.enableLocalDatastore()`. Then the Table View (second TabBarIcon) will never stop loading or call objectsDidLoad like you can see in the logs.

If you've problems to build, I will make a screencast :) .

**Comment by nlutsenko-fb** [comment]
@flashspys After checking out your repo - I think I found the problem...
You are using `ParseUI 1.0.2`, whilst the latest version is `1.1.0`.
(there was a bug with Local Datastore, that was fixed in `1.0.3`)

Just by replacing ParseUI.framework in your project with 1.1.0 - everything magically works for me now ;)
To avoid these kinds of problems (missing upgrade), I would suggest trying out Cocoapods.

Thanks for the feedback! Keep it coming!
